custom/icds/location_reassignment/tests/test_dumper.py

```diff
@@ -0,0 +1,69 @@
+from unittest import TestCase
+
+from mock import patch
+
+from corehq.util.workbook_json.excel import get_workbook
+from custom.icds.location_reassignment.dumper import Dumper
+
+
+class TestDumper(TestCase):
```

Comment:
Could this be a `SimpleTestCase`? It looks like all the database calls are mocked.

Reply:
I have used the `unittest.TestCase` instead of the django one for all tests for location reassignment that does not allow db calls.